types/src/toGamut.d.ts

```diff
@@ -1,24 +1,45 @@
 import { ColorTypes, PlainColorObject } from "./color.js";
+import { Methods } from "./deltaE/index.js";
 import ColorSpace, { Ref } from "./space.js";
 
+export interface Options {
+	/**
+	 * How to force into gamut.
+	 *
+	 * If `"clip"`, coordinates are just clipped to their reference range.\
+	 * If `"css"`, coordinates are reduced according to the CSS 4 Gamut Mapping Algorithm.\
+	 * If in the form `[colorSpaceId].[coordName]`, that coordinate is reduced
+	 * until the color is in gamut. Please note that this may produce nonsensical
+	 * results for certain coordinates (e.g. hue) or infinite loops
+	 * if reducing the coordinate never brings the color in gamut
+	 */
+	// eslint-disable-next-line @typescript-eslint/ban-types
+	method?: "css" | "clip" | (string & {}) | undefined;
```

Comment:
FYI, the `string & {}` type here is done for intellisense while writing code. Doing this means that editors will automatically suggest `"css"` and `"clip"` while typing, but any string type will still be accepted. If the type were just `string` instead, then the entire type would be simplified into `string | undefined` and the intellisense wouldn't be as helpful.

Reply:
Whoa, TIL 🎉

Reply:
Should we note that this uses `css` by default?

Reply:
Probably not a bad idea. I'll add that when I get a chance